CMakeLists.txt

```diff
@@ -396,6 +410,43 @@ install(TARGETS KVEngine Launcher
 install(FILES ${CMAKE_BINARY_DIR}/metadata.json
         DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/${EXEC_DIR}/)
 
+install(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS}
+        DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/${EXEC_DIR}/
+        COMPONENT Applications)
+
+IF(DEFINED ENV{APPVEYOR_BUILD_FOLDER}) # Are we building on Appveyor?
```

Comment:
If something is done only in AppVeyor then it is better to just move it in `appveyor.yml`.
Anyway, the proper installing is the different task.